starter-code/index.html

```diff
@@ -1,31 +1,47 @@
 <!DOCTYPE html>
+<!DOCTYPE html>
 <html lang="en">
 <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./css/main.css">
   <title>Instagram</title>
 </head>
 <body>
-  Instagram
-
-  Capture and Share the World's Moments
-
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
-
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-
-  Oh yeah, did we mention it's free?
-
-  Download on the App Store
-  Get it on Google Play
-
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
-
-  &copy; 2014 Instagram
+  <div>
+    <section>
+        <img src="images/phones.png" alt="huge phone"/>
+    </section>
+    <section class="right">
+      <header>
+        <a href="#"><img src="./images/brand.png" alt="instagram brand"/></a>
+        <button class="log-in"><img src="./images/home.png" alt="home"/>Log in</button>
+      </header>
+      <section class="middle">
+        <div class="text">
+            <h2>Capture and Share<br>the World's Moments</h2>
+            <p>Instagram is a <b>fast</b>, <b>beautiful</b> and <b>fun</b> way to share your life with friends and family.</p>
+            <p>Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.</p>
+            <p>Oh yeah, did we mention it's free?</p>
+        </div>
+      </section>
+      <section class="app">
+          <button class="appbutton"><img src="./images/badge-iphone.png" alt="app store"/></button>
+          <button class="appbutton"><img src="./images/badge-android.png" alt="play store"/></button>
+      </section>
+    </section>
+  </div>
```

Comment:
well done! 

Flex boxes seem to be understood! I would resize the right box as it's looking bigger than the original. For the rest it's perfect!